docs/api/rasa-sdk.rst

```diff
@@ -0,0 +1,90 @@
+:desc: Information about `rasa-sdk` objects and functions.
+
+.. _rasa-sdk:
+
+Rasa SDK
+========
+
+.. edit-link::
+
+Rasa SDK provides the tools you need to write :ref:`custom-actions`.
```

Comment:
```suggestion
Rasa SDK provides the tools you need to write :ref:`custom-actions` in Python.
```

Reply:
Can we add a quick sentence about the sdk having a few relevant classes to action-writing to lead into contents, since the contents list is currently a list of classes?